.env.example

```
@@ -412,6 +412,21 @@ ANTHROPIC_API_KEY=sk-ant-your-key-here
 #
 # Usage limits are displayed via /usage command.
 
+
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)